bubuku/features/metric_collector.py

```diff
@@ -23,14 +23,20 @@ def __init__(self, zk: BukuExhibitor):
 
     async def _get_metrics_from_broker(self, broker_id: int):
         broker_address = self.zk.get_broker_address(broker_id)
-        data = {'broker_address': broker_address, 'broker_id': broker_id}
+        data = {'broker_address': broker_address, 'broker_id': broker_id, 'metrics': {}}
         for metric in self.get_metric_mbeans():
-            request = requests.get("http://{}:{}/jolokia/read/{}".format(
+            metric_fetched = False
+            response = requests.get("http://{}:{}/jolokia/read/{}".format(
```

Comment:
could throw an exception, please wrap it with except

Reply:
for example, instance is restarting, and host is not available -> connection error